module-2/backend/week_two.md

```diff
@@ -26,21 +37,48 @@ end
 
 Now how would you find the owner of the team with an id of 4?
 
+team = Team.find(4)
+team.owner
+
 3. What do they allow you to do?
+I'm confused. Who are they?
 7. In a database that's holding students and teachers, what will be the relationship between students and teachers? Draw the schema diagram.
+
+teacher (n) |has| (n) student
+
 8. Define foreign key, primary key, and schema.
+Primary key is the unique identifier for a row. 
+Foreign key is the primary key of a row in another table that is linked to the current table.
+Schema is the overall database design.
+
 9. Describe the relationship between a foreign key on one table and a primary key on another table.
+The primary key of a table is the foreign key on the table it is linked to. Ex: Student - student_id (primary key), name, teacher_id
+
 10. What are the parts of an HTTP response?
+I don't really understand. Do you mean params? 
```

Comment:
An HTTP response has headers and a body. The status code (e.g. 404, 200, 503) is the first header.